README.md

```diff
@@ -0,0 +1,415 @@
+<div align="center">
+<img alt="Redoc logo" src="doc/images/vecteezy_cat-with-book-studying-logo-mascot-character-design_23286095.png"
+width="200px" />
+
+# Online Bookstore
+
+</div>
+
+## Introduction
+
+Greetings! Allow me to introduce you to the Online Bookstore app.
+This project encompasses the essential functionalities needed to operate a real-time bookstore, such as:
+  
+  User registration and authorization
+
+  User shopping cart: includes functional from filling the cart with products to creating an order
+
+  Creating and managing books available in the store and orders by store personnel
+
+# Why Online Bookstore?
+
+This project will appeal to individuals who need a simple and efficient tool to solve business problems
+without being burdened by unnecessary features.
+It does not require special training for users,
+whether they are store personnel or customers.
+Features
+
+# How It Works
+
+Video - //TODO
+
+Thanks to the integrated Liquibase, administrators do not need to manually create tables and relationships.
+They only need to create the database and then populate it with new products.
+Users need to register or log in by sending the appropriate request.
+After authorization, users can use their JWT token or registration details to send requests,
+add desired items to their cart by sorting books by categories or from the general list
+and create an order.
+As the order is processed, the administrator updates the order status,
+allowing the user to track the progress of their order.
+In short here is the list of features it can do
+
+    1. User Registration and Authentication
+    2. Book Browsing and Filtering by Categories
+    3. Shopping Cart Management - add, update, delete books in cart
+    4. Order Processing - create order using items in cart, check users orders
+    5. Admin Dashboard for Managing Books, Book Categories and Orders
+    6. Secure RESTful API Endpoints
+
+
+# Installation
+## Setup Instructions
+
+Clone the repository:
+
+    git clone git@github.com:azzilian/onlinebookstore.git
+
+Build the project using Maven:
+
+    mvn clean install
+
+Configure the database: Create a new MySQL database:
+
+    CREATE DATABASE bookstoreDB;
+
+Update the application.properties file with your MySQL database credentials (Usage of environment variables recommended):
+
+    spring.datasource.url=jdbc:mysql://localhost/bookstoreDB?serverTimezone=UTC
+    spring.datasource.username=${USERNAME}
+    spring.datasource.password=${PASSWORD}
+    spring.jackson.deserialization.fail-on-unknown-properties=true
+    spring.datasource.driver-class=com.mysql.cj.jdbc.Driver
+    spring.jpa.hibernate.ddl-auto=validate
+    spring.jpa.show-sql=true
+    jwt.expiration=${EXPIRATION}
+    jwt.secret=${SECRETKEY}

```

Comment:
let's just suggest to create `.env` file right away with example how it may look, instead of modifying the app.properties file